lib/saml_idp/xml_security.rb

```diff
@@ -237,6 +236,13 @@ def verify_signature(base64_cert, sig_alg, signature, canon_string, soft)
         cert = OpenSSL::X509::Certificate.new(cert_text)
         signature_algorithm = algorithm(sig_alg)
 
+        if signature_algorithm != OpenSSL::Digest::SHA256
+          return false if soft
+
+          raise ValidationError.new('Signature Algorithm needs to be SHA256',
+                                                           :wrong_sig_algorithm)
```

Comment:
personal style, I'd change the layout here
```suggestion
          raise ValidationError.new(
            'Signature Algorithm needs to be SHA256',
            :wrong_sig_algorithm,
          )
```

Reply:
updated in https://github.com/18F/saml_idp/pull/124/commits/8658a88a0e129e2a06f451b4e63e65c2dffa09ce